src/Orleans.Reminders/ReminderService/LocalReminderService.cs

```diff
@@ -1,21 +1,30 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 using Orleans.CodeGeneration;
+using Orleans.Hosting;
 using Orleans.Internal;
+using Orleans.Reminders;
+using Orleans.Runtime;
+using Orleans.Runtime.ConsistentRing;
+using Orleans.Runtime.Scheduler;
 
 namespace Orleans.Runtime.ReminderService
 {
-    internal sealed class LocalReminderService : GrainService, IReminderService
+    internal sealed class LocalReminderService : SystemTarget, IReminderService, ILifecycleParticipant<ISiloLifecycle>, IRingRangeListener
```

Comment:
I probably told you to implement SystemTarget directly instead of implementing GrainService - but I regret that: we should try to keep foreign changes to a minimum

Reply:
I've pushed an update where I've attempted to rectify that